src/TraceEvent/ZippedETL.cs

```diff
@@ -358,13 +357,13 @@ internal static List<string> GetNGenPdbs(string etlFile, SymbolReader symbolRead
     }
 #endif
 
-            /// <summary>
-            /// ZippedETLReader is a helper class that unpacks the ZIP files generated
-            /// by the ZippedETLWriter class.    It can be smart about placing the 
-            /// symbolic information in these files on the SymbolReader's path so that
-            /// symbolic lookup 'just works'.  
-            /// </summary>
-        public class ZippedETLReader
+    /// <summary>
```

Comment:
I would like to fix this differently.    We are going to make a new class ZippedDataReader and we are going to mark ZippedETLReader as obsolete.    We will implement ZippedETLReader by inheriting from ZippedDataReader.  

ZippedDataReader should have a property DataFileName, and should not have wired in dependencies.  It should simply look for the first file at the top level as the data file (and we can have another property called 'Extension' which allows us to look only for a file of a particular suffix).     

Now the Netperf case can use the ZippedDataReader without wired in changes.

Reply:
Synced offline. I'll leave this out from this effort and handle it from the callers' end. After all, I agree that it is a misuse for unzipping the netperf file using the ZippedETLReader.